### PR TITLE
ds course link

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by caupolicandiaz** [comment]
Created a new pr for a course link, the first pr was closed because the link was added to the wrong page

File: content/pandas/pandas.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Description: 'Pandas is a popular open-source Python module, used for data analy
 Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/python'
 CatalogContent:
   - 'learn-python-3'
+  - 'getting-started-with-python-for-data-science'
   - 'paths/data-science'
```
Comment (Dusch4593) [inline content/pandas/pandas.md:8]:
```suggestion
  - 'getting-started-with-python-for-data-science'
  - 'paths/data-science'
```

I agree that this new link is more relevant to pandas as a topic. Let's keep this to two slugs, though. (one free slug and one paid course slug)

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Thanks for adding this, @caupolicandiaz! I only had one comment/suggestion. After that, this should be good to merge. 😄

**Comment by caupolicandiaz** [comment]
@Dusch4593 should be all set now. Thanks

**Review by Dusch4593 — APPROVED** [review]
Thanks, @caupolicandiaz! Merging..

**Pull request merged**